Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG IMAGE_REF=node:12.21.0-alpine3.12
+
+# stage-0: install dependencies
+FROM $IMAGE_REF as stage-0
+WORKDIR /usr/src/app/
+COPY package.json package-lock.json ./
+RUN npm install
+
+# final stage
+FROM $IMAGE_REF
+WORKDIR /usr/src/app/
+COPY --from=stage-0 /usr/src/app/node_modules/ ./node_modules/
+COPY .babelrc api-docs.js api-key-auth.js index.js logger.js version.js package.json package-lock.json utils/ ./
```

Comment:
Do we need all of those (maybe `.babelrc` is the one that looks like isn't doing much to me)?

I just noticed we have a lot of loose source files, we could use a refactor to put them on a `src` dir or something.

Also I think it doesn't look like `stage-0` is saving any space. I think a single stage would be fine since we don't have any build dependencies.

Reply:
We need `.babelrc` so `npm start` runs properly. I just did the refactor and copied the `src/` dir. I also removed the stage-0. Can you review again, please?

Reply:
Got it. Looks great!